apps/api/src/applicant/ienapplicant.util.service.ts

```diff
@@ -245,7 +245,6 @@ export class IENApplicantUtilService {
         // Let's not put hard limit right now, and accept multiple records here.
         if (previousStatus.length > 0) {
           const updateData = {
-            updated_by: data.added_by,
```

Comment:
It causes IEN-352 issue. It could be reasonable to update the previous one's updated_by because it updates end_date. But it might mislead users who don't know this background process, so I deleted it.

Reply:
That is reasonable, Let's skip it.